src/pytorch_lightning/core/module.py

```diff
@@ -1479,6 +1479,34 @@ def clip_gradients(
             gradient_clip_val: The value at which to clip gradients.
             gradient_clip_algorithm: The gradient clipping algorithm to use. Pass ``gradient_clip_algorithm="value"``
                 to clip by value, and ``gradient_clip_algorithm="norm"`` to clip by norm.
+
+        Note:
+            For manual optimization (``self.automatic_optimization = False``), if you want to use gradient clipping,
+            consider calling ``self.clip_gradients(opt, gradient_clip_val, gradient_clip_algorithm)`` as shown in the
+            example (with a single optimizer) below,
```

Comment:
I suggest to drop the note and instead adjust the descriptions above to keep this concise.

Reply:
What do you mean by adjust the description. Making it concise like this? Or after `Handles gradient clipping internally`?
```
"""Handles gradient clipping internally.

      Note:
            - Do not override this method. If you want to customize gradient clipping, consider using :meth:`configure_gradient_clipping` method.
            - For manual optimization (``self.automatic_optimization = False``), if you want to use gradient clipping, consider calling ``self.clip_gradients(opt, gradient_clip_val, gradient_clip_algorithm)`` manually in the training step.

        Args:
            optimizer: Current optimizer being used.
            gradient_clip_val: The value at which to clip gradients.
            gradient_clip_algorithm: The gradient clipping algorithm to use. Pass ``gradient_clip_algorithm="value"``
                to clip by value, and ``gradient_clip_algorithm="norm"`` to clip by norm.
        """
```

Reply:
Done.